fast64_internal/f3d/occlusion_planes/exporter/to_c/occlusion_planes.py

```diff
@@ -0,0 +1,29 @@
+from mathutils import Vector
+
+from ...exporter import OcclusionPlaneCandidate, OcclusionPlaneCandidatesList
+from .....utility import CData
+
+
+def occVertexToC(vertex: Vector):
+    return "\t\t\t{" + ", ".join([str(int(round(a))) for a in vertex]) + "},\n"
```

Comment:
now we avoid using tabs for exported stuff, instead you can use the ``indent`` variable from ``utility.py`` (defined as ``indent = " " * 4``)

Reply:
Fixed.